### PR TITLE
[MIBM-511] add 'is EU' to declaration confirmation page

### PR DESCRIPTION
**Pull request by andy1138**

### CODE REVIEW

File: app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationConfirmationView.scala.html [hunk 1]
```diff
@@ -81,6 +82,16 @@ <h2 id="goodsDetailsId" class="govuk-heading-m" id="main-message">@messages("che
      <dt id="@{s"quantityLabel_$index"}" class="govuk-summary-list__key">@messages("checkYourAnswers.detailsOfTheGoods.quantity")</dt>
      <dd id="@{s"quantity_$index"}" class="govuk-summary-list__value" style="text-transform: capitalize">@goods.categoryQuantityOfGoods.quantity</dd>
     </div>
+
+    @goods match {
+     case good:ImportGoods => {
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationConfirmationView.scala.html:87]: any reason why we are not showing  any row in-case of exports ?

File: conf/messages.en [hunk 2]
```diff
@@ -412,6 +412,7 @@ checkYourAnswers.detailsOfTheGoods.category = Type of goods
 checkYourAnswers.detailsOfTheGoods.quantity = Number of items
 checkYourAnswers.detailsOfTheGoods.vatRate = VAT rate
 checkYourAnswers.detailsOfTheGoods.country = Country
+checkYourAnswers.detailsOfTheGoods.producedInEu = Produced in EU
```
Comment (sureshhmrc) [inline conf/messages.en:415]: going forward we don't need to keep the above `checkYourAnswers.detailsOfTheGoods.country` key, i think

Comment (jordanrowe) [reply conf/messages.en:415]: country is still used for export

File: app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationConfirmationView.scala.html (hunk 1 above)
Comment (andy1138) [reply app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationConfirmationView.scala.html:87]: The base class `Goods` is missing  `producedInEu` so we cannot use that.

**Review by sureshhmrc — APPROVED** [review]

**Pull request merged**